services/web-client/source/class/qxapp/desktop/PrjEditor.js

```diff
@@ -30,32 +48,39 @@ qx.Class.define("qxapp.desktop.PrjEditor", {
       right: 0,
       bottom: 0
     });
-
     this.__showSettings(false);
 
-    this.__SettingsView.addListener("SettingsEditionDone", function() {
+    this.__SettingsView.addListener("SettingsEditionDone", () => {
       this.__showSettings(false);
     }, this);
 
-    this.__Workbench.addListener("NodeDoubleClicked", function(e) {
+    this.__Workbench.addListener("NodeDoubleClicked", e => {
       this.__showSettings(true);
       this.__SettingsView.setNodeMetadata(e.getData());
     }, this);
+
+    this.__TransDeco = new qx.ui.decoration.Decorator().set({
+      transitionProperty: ["left", "right", "width"],
+      transitionDuration: "0.1s",
+      transitionTimingFunction: "ease"
+    });
   },
 
   members: {
     __Pane: null,
```

Comment:
didn't we agree on naming variables as ``__myVariable``? I wonder why this is not captured by the lintier? Please tune the ESlinter to catch all these conventions.

Reply:
yes by all means! __xyzAbc seems much nicer to me ...